cmd/otelcol/config/collector/ecs_ec2_config.yaml

```diff
@@ -125,12 +125,12 @@ exporters:
     access_token: "${SPLUNK_ACCESS_TOKEN}"
     realm: "${SPLUNK_REALM}"
     correlation:
-#  # Logs
-#  splunk_hec:
-#    token: "${SPLUNK_HEC_TOKEN}"
-#    endpoint: "${SPLUNK_HEC_URL}"
-#    source: "otel"
-#    sourcetype: "otel"
+  # Logs + Profiling
+  splunk_hec:
+    token: "${SPLUNK_HEC_TOKEN}"
+    endpoint: "${SPLUNK_HEC_URL}"
```

Comment:
these env vars aren't documented in the deployment instructions, and if they are now intended to be used for profiling we probably want to have a templated endpoint like we do w/ the sapm exporter.

Reply:
> these env vars aren't documented in the deployment instructions, and if they are now intended to be used for profiling we probably want to have a templated endpoint like we do w/ the sapm exporter and use SPLUNK_ACCESS_TOKEN*

We [compose these env vars](https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/main.go#L365-L388) from `SPLUNK_ACCESS_TOKEN` and `SPLUNK_REALM` by default. Using these env vars leaves an option for users to send logs to another Splunk endpoint instead of o11y without providing a custom config. Also it's consistent with other configs that we have

> imo we should have a dedicated splunk_hec/profiling exporter.

The profiling data shares the same pipeline including otlp receiver. Profiling libraries use the same OTLP port by default. So we cannot split it and create another exporter.

Reply:
I think the autosetting pass is ok where users aren't setting these env vars for splunk cloud usage. Otherwise wouldn't profiling data be sent directly to Splunk Cloud?

Reply:
> Otherwise wouldn't profiling data be sent directly to Splunk Cloud?
Yes they would, and Splunk Cloud backend can digest and show them is some way, not as good as Splunk O11y tho.

Use case when user wants logs flowing to Splunk Cloud and Profiling data to Splunk O11y will require separate pipelines and changing logs and profiling libraries to send to different ports. So it using `SPLUNK_ACCESS_TOKEN` instead of `SPLUNK_HEC_TOKEN` doesn't help. 

I think we should use `SPLUNK_HEC_TOKEN` at least for consistency with other default configs.

Reply:
@rmfitzpatrick please let us know if you're ok to merge it